mocks/mock_OnMessage.go

```diff
@@ -0,0 +1,74 @@
+// Code generated by mockery v2.42.1. DO NOT EDIT.
```

Comment:
@shan-96 this file  doesn't look to be used anywhere

Reply:
you're right, it got generated when I ran `make mocks`.
Will remove it